src/containers/BooksLists.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+// import { useState } from 'react';
+// import { connect } from 'react-redux';
+
+export default function BooksList({ book }) {
+  return (
+    <div className="table-div">
+      <table>
+        <tr>
+          <th>Book Id</th>
+          <th>Title</th>
+          <th>Author</th>
+          <th>Category</th>
+        </tr>
+        <tr>
+          <td>{book.id}</td>
+          <td>{book.title}</td>
+          <td>{book.author}</td>
+          <td>{book.category}</td>
+        </tr>
+      </table>
+    </div>
+  );
+}
+
+BooksList.propTypes = {
+  book: PropTypes.instanceOf(Object),
+};
+
+BooksList.defaultProps = {
+  book: {
+    id: '1',
+    title: 'Fly',
+    author: 'JK',
+    category: 'Mistery',
+  },
+};
```

Comment:
-  [MANDATORY] BooksList component ...
    - [ ]  Should be connected to redux store. 
    - [ ]  Should receive a list of books from props. not just single book.
    - [ ]  The default book list should be coming from the store in initial state.
    - [ ] The table should iterate through the list of books and display them.